components/brave_wallet_ui/components/opt-in/index.tsx

```diff
@@ -47,8 +45,10 @@ export default class OptIn extends React.PureComponent<Props, State> {
   }
 
   onWalletOptIn = () => {
-    this.setState({ isLoading: true })
-    this.props.onWalletOptIn()
+    if (!this.state.isLoading) {
```

Comment:
This didn't cause problems that I could see, but it's a good idea to avoid double opt-in